### PR TITLE
Merging ZETA code and config to the main branch

### PR DESCRIPTION
**Pull request by rohitshubham**
Next will work on some more cleanup and documentation

### CODE REVIEW

**Comment by linhsolar** [comment]
first, i think it is not good to have ZETA/zera in the structure. you might think to have 
ZETA/[subdir]
ZETA/src/[modules]

instead of ZETA/zeta/src (everything under ZETA is about ZETA anyway)

File: scripts/k6s-auth-token-stress-test-script/script.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+export let options = {
+  stages: [
+    { duration: '1s', target: 40 },
+   // { duration: '1m', target: 1 },
+    { duration: '1m', target: 40 },
+  ],
+};
+
+export default function () {
+  let res = http.get('http://195.148.21.10:5000/authentication?name=test-service&capabilities=cpu&expiry=1');
```
Comment (linhsolar) [inline scripts/k6s-auth-token-stress-test-script/script.js]: a hard code here

**Comment by rohitshubham** [comment]
> first, i think it is not good to have ZETA/zera in the structure. you might think to have
> ZETA/[subdir]
> ZETA/src/[modules]
> 
> instead of ZETA/zeta/src (everything under ZETA is about ZETA anyway)

Made the directory structure change and the build change too

**Comment by rohitshubham** [comment]
@linhsolar

**Pull request merged**